Common/Models/User/UserGroupModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Postgrest.Attributes;
+
+namespace Common.Models.User
+{
+    [Table("UsersGroups")]
+    public class UserGroupModel : BaseDbModel
+    {
+        [PrimaryKey("Id")]
+        public int Id { get; set; }
+
+        [Column("AmountOfUsers")]
+        public int AmountOfUsers { get; set; }
+
+        public virtual ICollection<UserModel> Users { get; set; } = new HashSet<UserModel>();
```

Comment:
Are we having a collections inside a model? That does not sound very safe either. Model should only describe basic properties of the entity and SQL entity cannot hold complex structures such as collections no? Does not sound very intuitive frankly.